theme_artists/views/snippets/s_call_to_action.xml

```diff
@@ -8,7 +8,7 @@
         </xpath>
         <!-- Shape -->
         <xpath expr="//div[hasclass('container')]" position="before">
-            <div class="o_we_shape o_web_editor_Airy_12 o_we_flip_y"/>
+            <div class="o_we_shape o_web_editor_Airy_12 o_we_flip_y o_footer_extra_shape_mapping"/>
```

Comment:
This should not be done, the point is that this class is automatically added when building pages **through the new website configurator**. When using the editor, you don't want that class when dropping a snippet into the page, otherwise the shape would be linked to the footer color wherever it is dropped.

Reply:
Ok I thought it was an issue, but yeah it's logical in fact :). I removed the [FIX] commit.